scripts/cue-publish/cue-publish.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/perses/plugins/scripts/tag"
+	"github.com/sirupsen/logrus"
+)
+
+const modulePrefix = "github.com/perses/plugins"
+
+func runCommand(name string, args ...string) error {
+	cmd := exec.Command(name, args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("failed to run %s %v: %w\nstderr: %s", name, args, err, stderr.String())
+	}
+	return nil
+}
+
+func main() {
+	token := flag.String("token", "", "Authentication token for CUE Central Registry login")
+	t := tag.Flag()
+	flag.Parse()
+
+	if *token == "" {
+		logrus.Fatal("Error: -token flag is required")
+	}
+	if *t == "" {
+		logrus.Fatal("Error: -tag flag is required")
+	}
+
+	pluginName, version := tag.Parse(t)
+	module := fmt.Sprintf("%s/%s@%s", modulePrefix, pluginName, version)
```

Comment:
I thought the version should be prefixed by `v`. If it's the case, it is missing here then as the version returned by `tag.Parse` doesn't not contain the `v`